### PR TITLE
OPENDNSSEC-803: Crash reporting

### PR DESCRIPTION
**Pull request by halderen**

### CODE REVIEW

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
Ready to merge, finally.

File: signer/src/ods-signerd.c [hunk 1]
```diff
@@ -190,6 +197,7 @@ main(int argc, char* argv[])
     /* main stuff */
     fprintf(stdout, "OpenDNSSEC signer engine version %s\n", PACKAGE_VERSION);
 
+    ods_janitor_initialize(argv0);
```
Comment (yschaeff) [inline signer/src/ods-signerd.c]: On proper exit we leak argv0? If it is intentional please document here.

**Comment by yschaeff** [comment]
@halderen Done.

**Comment by halderen** [comment]
#build

**Pull request merged**